packages/plugin-dom-editable/src/EventNormalizer.ts

```diff
@@ -610,11 +617,25 @@ export class EventNormalizer {
             // TODO: no rendering in editable can happen before the analysis of
             // the selection. There should be a mechanism here that can be used
             // by the normalizer to block the rendering until this resolves.
+            const currentDomSelection = this._getSelection();
             this._keyboardSelectionTimeout = new Timeout<EventBatch>(
                 async (): Promise<EventBatch> => {
+                    const newDomSelection = this._getSelection();
+                    const collapsed =
+                        newDomSelection.anchorNode === newDomSelection.focusNode &&
+                        newDomSelection.anchorOffset === newDomSelection.focusOffset;
+
+                    let domSelection;
+                    if (collapsed && !this._isInEditable(newDomSelection.anchorNode)) {
+                        // Prevent a collapsed selection outside the editable
+                        // context: keep the current selection.
+                        domSelection = currentDomSelection;
+                    } else {
+                        domSelection = newDomSelection;
+                    }
```

Comment:
I am not satisfied.

If a node is not editable and the normalizer send an action for that node (like setSelection), it is the role of the editor to discard the normalized event, not the normalizer itself.
The normalizer normalize a contenteditable without any knowlede about any other structure.

Reply:
I merely _corrected_ the current behavior which _did_ already prevent the action right from the `EventNormalizer` through `isInEditable` checks, but did it wrong in some cases.
I remember this is an old debate we had like 6 months ago. I think chm had introduced those checks so that the browser would revert to default outside the editable zones.
If we want to _change_ the current behavior I'm not necessarily opposed to it but it's definitely out of scope, especially in the current rush.

Reply:
Ok.